docs/Comparison.md

```diff
@@ -2,23 +2,24 @@
+AlmaLinux utilizes package sources from both RHEL and CentOS Stream to build its distribution. In contrast, Oracle Linux and Rocky Linux rely exclusively on package sources from RHEL.
+
+|Benchmarking against RHEL       |AlmaLinux                                 |Oracle Linux                        |Rocky Linux                                                           |CentOS Stream                  |RHEL                           |
+|--------------------------------|------------------------------------------|------------------------------------|----------------------------------------------------------------------|-------------------------------|-------------------------------|
+|First release                   |Since March 2021                          |Since 2006                          |Since June 2021                                                       |Since 2019                     |March 2002                     |
+|RHEL compatibility              |minor version compatible                  |minor version compatible            |minor version compatible                                              |major version compatible       |n/a                            |
+|Last minor version release delay|[5 days][alma9.4]                         |[6 days][oracle9.4]                 |[8 days][rocky9.4]                                                    |N/A                            |n/a                            |
+|Errata                          |Yes                                       |Yes                                 |Yes                                                                   |No                             |yes                            |
+|Lifecycle                       |10 Years                                  |10 Years                            |10 Years                                                              |5.5 Years                      |10-14 years                    |
+|Commercial support              |3rd party                                 |Oracle, 3rd parties                 |3rd parties                                                           |3rd Party                      |1st party (Red Hat)            |
+|Livepatching service            |KernelCare                                |Oracle Ksplice, KernelCare Available|KernelCare Available                                                  |Not available                  |kpatch                         |
+|Architectures                   |aarch64, ppc64le, s390x                   |aarch64                             |aarch64, ppc64le (versions ≥ 9), s390x (versions ≥ 9)                 |aarch64, ppc64le               |aarch64, ppc64le, s390x, x86_64|
+|Backing organization            |AlmaLinux OS Foundation (501c6 non-profit)|Oracle Inc (For Profit C-Corp)      |Rocky Enterprise Software Foundation (For Profit, Public Benefit Corp)|Red Hat Inc (For Profit C-Corp)|Red Hat Inc.                   |
```

Comment:
I would drop the "(For Profit C-Corp)" since the important part of this comparison it to call out that Alma is backed by a non-profit.

Reply:
Done!

Reply:
This line still says "for profit" in two spots.  I would also drop the C-Corp part, as it's implied by the "Inc."